README.md

```diff
@@ -110,7 +110,8 @@ console.log('curl -i http://localhost:3000/users -d "name=test"');
 - `includeUnparsed` **{Boolean}** Toggles co-body returnRawBody option; if set to true, for form encodedand and JSON requests the raw, unparsed requesty body will be attached to `ctx.reqeust.body` using a `Symbol`, default `false`
 - `formidable` **{Object}** Options to pass to the formidable multipart parser
 - `onError` **{Function}** Custom error handle, if throw an error, you can customize the response - onError(error, context), default will throw
-- `strict` **{Boolean}** If enabled, don't parse GET, HEAD, DELETE requests, default `true`
+- `strict` **{Boolean}** ***DEPRECATED*** If enabled, don't parse GET, HEAD, DELETE requests, default `true`
+- `parsedMethods` **{String[]}** If specified, declares the methods where bodies will be parsed, default `['POST', 'PUT', 'PATCH', 'DELETE']`
```

Comment:
I would change the language to read

Declares the HTTP methods where bodies will be parsed, default `['POST', 'PUT', 'PATCH', 'DELETE']`. Replaces `strict` option.

Reply:
Disregard. We reset the "test database" before each test.

Reply:
Sure.  Also, forgot that I took DELETE out of the defaults; I need to update that as well.  Give me a few ticks and I'll change & push.

Reply:
Adjustment made per recommendation.